### DISCUSSION TITLE
When entity count is 1 and no Wikipedia or Fandom

### DISCUSSION THREAD

**Discussion by jps-e** [xxyzz/WordDumb#38] [Q&A] [answered]
When nothing is found for an entity in Wikipedia or Fandom a context quote is used for the description. In cases when there are multiple occurrences this is good. But when the entity only occurs once in the book this just clutters X-ray. It might be that this isolated occurrence is an alias for another character so it still might be useful to have these single occurrence entities found by WordDumb.

But after these have been fixed using worddumb-custom-x-ray.json I think WordDumb should have a cusomize option to set a minimum number counts to qualify for an X-ray entry and no X-ray entry made for entities with a count below the minimum. Otherwise there might be hundreds of entities with a count of 1 and no description from Wikipedia or Fandom.

Of course any entity in worddumb-custom-x-ray.json would be exempt from this minimum.

(I made the category Q&A instead of Ideas so that answers are enabled.)

**Comment by xxyzz** [comment, marked as answer]
Please try the plugin from https://github.com/xxyzz/WordDumb/actions/runs/2600240882 when the test job is finished.

**Reply by jps-e** [reply]
Thanks! It works.
